pkg/model/gcemodel/context.go

```diff
@@ -132,6 +132,14 @@ func (c *GCEModelContext) NameForFirewallRule(id string) string {
 	return gce.ClusterSuffixedName(id, c.Cluster.ObjectMeta.Name, 63)
 }
 
+func (c *GCEModelContext) NetworkIPAliasRange() *string {
+	if c.Cluster.Spec.CloudConfig.GCPAliasRange != nil {
+		return c.Cluster.Spec.CloudConfig.GCPAliasRange
+	}
```

Comment:
Are we validating that the string value is sensible anywhere?

Reply:
Thanks, I can add that!

Reply:
Nice!